### ISSUE TITLE
MXGP: The Official Motocross Videogame graphical issue

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#13099]
### Quick summary

the game has graphical issues which pop up when the camera faces certain angles.

### Details

when testing this game there are single frames where certain textures seem to break and stretch in weird ways.
Rpcs3
![NPEB90530_screenshot_2022_12_19_15_48_17](https://user-images.githubusercontent.com/71127352/208514158-c7c13bde-c123-4636-a536-dd187bb4b256.png)
![NPEB90530_screenshot_2022_12_19_15_49_07](https://user-images.githubusercontent.com/71127352/208514208-076a771a-de28-4047-b7b6-8c00fd7a6f79.png)
https://youtu.be/RxPm0VHIc1U?t=77

PS3
https://www.youtube.com/watch?v=Xg1goFzHe9w

### Attach a log file

[RPCS3MXGP Demo.log.gz](https://github.com/RPCS3/rpcs3/files/10262554/RPCS3MXGP.Demo.log.gz)

### Attach capture files for visual issues

https://www.dropbox.com/s/uy5wrsow0j1wbzt/NPEB90530_20221219154924_capture.zip?dl=0

### System configuration

    SYS: RPCS3 v0.0.25-14478-812224f7 Alpha | master
    SYS: AMD Ryzen 5 2600 Six-Core Processor | 12 Threads | 15.95 GiB RAM | TSC: 3.400GHz | AVX+ | FMA3
    SYS: Operating system: Windows, Major: 10, Minor: 0, Build: 19044, Service Pack: none, Compatibility mode: 0
    SYS: Current Time: 2022-10-28T21:53:46
    RSX: Found vulkan-compatible GPU: 'NVIDIA GeForce RTX 3060 Ti' running on driver 526.47.0.0
    SYS: Firmware version: 4.89

**Comment by kd-11** [comment]
Usually resolved using WCB in games that use SPU culling like this.

**Comment by Triticum0** [comment]
Tested with Write colour buffer enable and it didn't fix the issue on the savestate I'm using to test the issue on.

**Comment by Darkhost1999** [comment]
Can we regression test this with WCB. Similar behavior is observed in NFS Rivals where WCB used to resolve these graphics same as Battlefield 3 but neither of those 2 games are in a good enough condition to prove or disprove this issue.
If this game looking as good as it does can isolate the build that started WCB not fixing the issues maybe we can positively effect other games resolving this.

**Comment by Triticum0** [comment]
This issue also affects MX vs ATV Reflex

**Comment by Triticum0** [comment]
Retested and there no change. Using other buffers + Cpu Blits

**Comment by Triticum0** [comment]
@Darkhost1999 If you want to retest Battlefield 3 and NFS Rivals to see if the improved or related to this issue as tested both MX vs ATV Reflex and MXGP: The Official Motocross Videogame and both seem to be fixed

**Comment by Triticum0** [comment]
Note there is some z-fighting but move it to #13078

**Comment by Triticum0** [comment]
Also with more testing WCB now actually does something when comes to rendering dirt texture but need WCB + RCB or dirt causes graphcial issue and effects physic cause vehicle  to get suck at start line on MX vs ATV Reflex. doesn't seem that this game effect though.

**Comment by digant73** [comment]
yes, I think it's good to collect z-fighting comments directly in #13078.
If possible, please submit your findings on this games to wiki page so all your big effort is no lost.
thanks
